config/defaults.go

```diff
@@ -42,6 +42,7 @@ func DefaultConfig(home, chainId string) *NodeConfig {
 			BootstrapTime:           30 * time.Second,
 			ListenAddress:           DefaultListenAddress,
 			BootstrapNodes:          "",
+			Advertising:             true,
```

Comment:
Would prefer `Advertise` or `AdvertisingEnabled` as a name, ditto for p2p.go

Reply:
done